core/modules/utils/dom/modal.js

```diff
@@ -209,6 +210,9 @@ Modal.prototype.display = function(title,options) {
 	headerWidgetNode.addEventListener("tm-close-tiddler",closeHandler,false);
 	bodyWidgetNode.addEventListener("tm-close-tiddler",closeHandler,false);
 	footerWidgetNode.addEventListener("tm-close-tiddler",closeHandler,false);
+	if(options.maskClosable) {
```

Comment:
This only checks if the variable exists, but IMO it should also check for the value `=== true` 
I'm not sure. Is the maskClosable accessible from wikitext? .. If yes it should be "yes" instead of true. .. If it's only accessible with js functions true is OK

Reply:
Now I know it can...